spec/api/v1/rswag_spec.rb

```diff
@@ -57,7 +57,7 @@
 
     post "Update the Complexity of Need level for an offender" do
       tags "Single Offender"
-      description "Clients calling this endpoint must have role: `ROLE_COMPLEXITY_OF_NEED` with scope: `write`"
+      description "Clients calling this endpoint must have role: `ROLE_UPDATE_COMPLEXITY_OF_NEED`"
```

Comment:
We should also update the role & scope requirements documented in the top-level API description.

See `spec/swagger_helper.rb`:

https://github.com/ministryofjustice/hmpps-complexity-of-need/blob/15ed5c0bc236bb3a14b0c5c394485118e74be673/spec/swagger_helper.rb#L31-L33

Which appears at the top of the Swagger UI page:

<img width="1278" alt="Screenshot 2021-04-19 at 14 05 21" src="https://user-images.githubusercontent.com/7735945/115241114-5d356700-a118-11eb-9b75-00b483cda9c8.png">

---

Whilst you're there, could you also change the Slack channel link?

We're in the **#manage-pom-cases** channel now:

```
[#manage-pom-cases](https://mojdt.slack.com/channels/manage-pom-cases)
```